RoutesApp_ios/repository/local/managers/CoreDataManager.swift

```diff
@@ -41,8 +41,28 @@ open class CoreDataManager {
     func getContext() -> NSManagedObjectContext {
         return persistentContainer.viewContext
     }
+    func updateDataValue(entity: String, key: String, keyValue: String, keyUpdate: String, keyUpdateValue: Any) {
+        let managedContext = persistentContainer.viewContext
+        let entity = NSEntityDescription.entity(forEntityName: entity, in: managedContext)
+               let request = NSFetchRequest<NSFetchRequestResult>()
+               request.entity = entity
+               let predicate = NSPredicate(format: "(\(key) = %@)", keyValue)
+               request.predicate = predicate
+               do {
+                   let results = try managedContext.fetch(request)
+                   guard let objectUpdate = results[0] as? NSManagedObject else {return}
+                   objectUpdate.setValue(keyUpdateValue, forKey: keyUpdate)
+                   do {
+                       try managedContext.save()
+                   } catch {
+                    print("error")
+                   }
+               } catch {
+                   print("error")
```

Comment:
instead of using print with no return value, maybe a completion escaping with void success and error. So we can control better on error result

Reply:
done:)